File: mRemoteV1/Messages/Messages.Message.cs
using System;


namespace mRemoteNG.Messages
{
	public enum MessageClass
	{
		InformationMsg = 0,
		WarningMsg = 1,
		ErrorMsg = 2,
		ReportMsg = 3
	}
		
	public class Message
	{
		private MessageClass _MsgClass;
        public MessageClass MsgClass
		{
			get
			{
				return _MsgClass;
			}
			set
			{
				_MsgClass = value;
			}
		}
			
		private string _MsgText;
        public string MsgText
		{
			get
			{
				return _MsgText;
			}
			set
			{
				_MsgText = value;
			}
		}
			
		private DateTime _MsgDate;
        public DateTime MsgDate
		{
			get
			{
				return _MsgDate;
			}
			set
			{
				_MsgDate = value;
			}
		}
	}
}

File: mRemoteV1/Messages/Messages.Message.vb
Namespace Messages
    Public Enum MessageClass
        InformationMsg = 0
        WarningMsg = 1
        ErrorMsg = 2
        ReportMsg = 3
    End Enum

    Public Class Message
        Private _MsgClass As MessageClass

        Public Property MsgClass As MessageClass
            Get
                Return _MsgClass
            End Get
            Set
                _MsgClass = value
            End Set
        End Property

        Private _MsgText As String

        Public Property MsgText As String
            Get
                Return _MsgText
            End Get
            Set
                _MsgText = value
            End Set
        End Property

        Private _MsgDate As Date

        Public Property MsgDate As Date
            Get
                Return _MsgDate
            End Get
            Set
                _MsgDate = value
            End Set
        End Property
    End Class
End Namespace